lib/protobuf/field/bool_field.rb

```diff
@@ -35,6 +35,21 @@ def decode(value)
         value == 1
       end
 
+      def define_decode_setter
+        field = self
+        field_name = field.name
+        name_method_name = "_protobuf_decode_setter_#{field.name}"
+        tag_method_name = "_protobuf_decode_setter_#{field.tag}"
+
+        message_class.class_eval do
+          define_method(name_method_name) do |val|
+            @values[field_name] = field.decode(val)
+          end
+
+          alias_method tag_method_name, name_method_name
```

Comment:
in `lib/protobuf/message/serialization.rb` looks like we only ever call the tag version of this method, why even define the field name versions at all?